docs/docusaurus/docs/cloud/set_up_gx_cloud.md

```diff
@@ -21,11 +21,7 @@ To get the most out of GX Cloud, you'll need to request a GX Cloud Beta account,
 
 ## Prepare your environment
 
-1. Download and install Python. See [Active Python Releases](https://www.python.org/downloads/).
-
-2. Download and install pip. See the [pip documentation](https://pip.pypa.io/en/stable/cli/pip/).
-
-3. Download and install Docker. See [Get Docker](https://docs.docker.com/get-docker/).
+1. Download and install Docker. See [Get Docker](https://docs.docker.com/get-docker/).
```

Comment:
Could we link to our agent repo for agent setup instructions?
Or at least have a line like "For alternative setup instructions refer to <AGENT_REPO_README>"